Aptoide/aptoide-client-v8
# Feature/asv 325 update popup dialog

**What does this PR do?**

This PR aims to change the share dialog after install when you are logged in to match the provided design. It also aggregates the metrics ticket as it was completely tied in with the work developed here

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AppViewInstallWidget.java

**How should this be manually tested?**

Try installing an app and click the 3 options to verify they are working appropriately. Skip should dismiss the dialog without sharing (I suggest you test this one first); Continue should dismiss the dialog and share the card; Don't show again should dismiss the card and not show it on subsequent install press, no matter the app

**What are the relevant tickets?**

  Tickets related to this pull-request: 
  [ASV-325](<https://aptoide.atlassian.net/browse/ASV-325>)
  [ASV-295](<https://aptoide.atlassian.net/browse/ASV-295>)

**Code Review Checklist**
